src/js/gi/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+class Breadcrumbs extends React.Component {
```

Comment:
We have a Breadcrumbs component in `rx` as well. Probably not for this PR, but it might be useful to pull both out into `common` at some point.